packages/maskbook/src/plugins/RedPacket/messages.ts

```diff
@@ -1,6 +1,12 @@
-import { RedPacketPluginID } from './constants'
+import { RedPacketPluginID, RedPacketNftPluginID } from './constants'
 import { createPluginMessage, createPluginRPC } from '@masknet/plugin-infra'
 
 if (import.meta.webpackHot) import.meta.webpackHot.accept()
 const RedPacketMessage = createPluginMessage(RedPacketPluginID)
+const RedPacketNftMessage = createPluginMessage(RedPacketNftPluginID)
 export const RedPacketRPC = createPluginRPC(RedPacketPluginID, () => import('./Worker/services'), RedPacketMessage.rpc)
+export const RedPacketNftRPC = createPluginRPC(
+    RedPacketNftPluginID,
+    () => import('./Worker/servicesForNft'),
+    RedPacketNftMessage.rpc,
```

Comment:
Please use choose another channel on `RedPacketMessage` instead of creating a new one.